<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadSensorsDialog</class>
 <widget class="QDialog" name="ReadSensorsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(152, 152, 152)}

QPushButton:hover{background-color: rgb(53, 161, 79);}

QPushButton:pressed{background-color: rgb(53, 161, 79)}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>451</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>1号传感器</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="file_1"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="open_1">
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(152, 152, 152)}

QPushButton:hover{background-color: rgb(53, 161, 79);}

QPushButton:pressed{background-color: rgb(53, 161, 79)}</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>2号传感器</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="file_2"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="open_2">
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(152, 152, 152)}

QPushButton:hover{background-color: rgb(53, 161, 79);}

QPushButton:pressed{background-color: rgb(53, 161, 79)}</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>3号传感器</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="file_3"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="open_3">
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(152, 152, 152)}

QPushButton:hover{background-color: rgb(53, 161, 79);}

QPushButton:pressed{background-color: rgb(53, 161, 79)}</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>4号传感器</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="file_4"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="open_4">
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(152, 152, 152)}

QPushButton:hover{background-color: rgb(53, 161, 79);}

QPushButton:pressed{background-color: rgb(53, 161, 79)}</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReadSensorsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReadSensorsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
